scrapy/extensions/feedexport.py

```diff
@@ -617,7 +617,7 @@ def _get_storage(self, uri, feed_options):
         It supports not passing the *feed_options* parameters to classes that
         do not support it, and issuing a deprecation warning instead.
         """
-        feedcls = self.storages[urlparse(uri).scheme]
+        feedcls = self.storages.get(urlparse(uri).scheme) or self.storages["file"]
```

Comment:
```suggestion
        feedcls = self.storages.get(urlparse(uri).scheme, self.storages["file"])
```

(assuming it works that way)

Reply:
I was wondering because i tried this before now i understand why did not work 🤡 

![image](https://github.com/scrapy/scrapy/assets/5853172/2b2f97e3-5905-4c00-9c8a-6bc00388efee)

Thanks @wRAR